providers/org/zenodo/harvester.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from furl import furl
+
+from share.harvest.oai import OAIHarvester
+
+logger = logging.getLogger(__name__)
+
+
+class ZenodoHarvester(OAIHarvester):
```

Comment:
We don't need it just yet, but a more extendable base class for OAI might prove to be useful.